### ISSUE TITLE
Actions set $name chinese then not work

### ISSUE THREAD

**Issue by loveyuxiao** [laravel/nova-issues#816]
When i use action in nova，i set the action name 
 public $name = '中文';
the action can`t be called

**Comment by yanhaovgo** [comment]
Same question. The play action button is disabled when the action name doesn't contain ascii. That means the play button is ok when the action name is '中国a' or 'a中国', but when name is '中国' or '한국' or 'こんにちは' the play button doesn't work well.

In the laravel/nova/src/Actions/Action.php:
```
    /**
     * Get the URI key for the action.
     *
     * @return string
     */
    public function uriKey()
    {
        // This method return '' when the name doesn't contain ascii.
        return Str::slug($this->name());
    }
```
And in the view laravel/nova/resources/js/components/ActionSelector.vue:
```
<button
    data-testid="action-confirm"
    dusk="run-action-button"
    @click.prevent="openConfirmationModal"

    <!-- This button is disabled cause selectedAction is always false --!>
    :disabled="!selectedAction"
     :class="{ 'btn-disabled' : !selectedAction }"

    class="btn btn-default btn-primary flex items-center justify-center px-3"
    title="Run Action">
    <icon type="play" class="text-white" style="margin-left: 7px" />
 </button>

computed: {
    // This property will always return false
    selectedAction() {
        if (this.selectedActionKey) {
            return _.find(this.allActions, a => a.uriKey == this.selectedActionKey)
        }
    },
}
```

**Comment by 907544030** [comment]
see https://github.com/laravel/nova-issues/issues/563

**Comment by alexbowers** [comment]
Closed as duplicate of #563
